templates/layouts/main-install.hbs

```diff
@@ -15,9 +15,9 @@
 
     {{!-- Content --}}
     <div class="container docs-container">
-      <p>Supported features of all watches are in the <a href="{{rel 'install/features'}}">Features Table</a></p>
-      <p>AsteroidOS can be installed as an alternative operating system on various smartwatches. Instructions for each model can be found below:</p>
-      <p>If you have questions regarding the installation process, please check out the <a href="#FAQ">Install FAQ</a> section.</p>
+      <p>AsteroidOS can be installed as an alternative operating system on various smartwatches. Instructions for each model can be found below.</p>
+      <p>Before installing AsteroidOS, make sure you are aware of the capabilities and limitations of AsteroidOS by consulting the <a href="{{rel 'faq'}}">FAQ page</a>.</p>
+      <p>In case you want to compare supported features of all watches AsteroidOS supports, have a look at the <a href="{{rel 'install/features'}}">Features Table</a>.</p>
```

Comment:
Maybe better would be to be more succinct here.  "A summary of the state of all features of all watches AsteroidOS supports is in the {features table hyperlink}"